meta-balena-dunfell/recipes-devtools/rust/rust%.bbappend

```diff
@@ -0,0 +1,64 @@
+# This task is defined in meta-rust. We over-ride it until
+# https://github.com/meta-rust/meta-rust/issues/240
```

Comment:
This issue is now closed so it needs to be replaced. If there is a version upstream that makes this unnecessary we should note it here.

Reply:
I found the upstream fix and confirmed our append can be removed for dunfell.